{{cookiecutter.project_slug}}/clients/mobile/react-native/src/services/user/api.ts

```diff
@@ -22,6 +22,7 @@ const requestPasswordResetCode = createCustomServiceCall(
     await client.get(`api/password/reset/code/${input.email}/`)
```

Comment:
Not sure if I've misread this but it looks like all of the other api calls have been changed to remove code. Should this not match that pattern too?

Reply:
ah this is a different endpoint to the one for the web views. In web we use the reset link, but in mobile we use a reset code. We found this approach to be a less cumbersome approach than trying to figure out and get `in-app` links working.